CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+#
+ogImage:
+twitterCreator: "@your_twitter_handle" # NOTE: The quotes are required!
+twitterDescription:
+twitterTitle:
+---
+```
+
+**NOTE:** The "ogImage" _must_ be a fully-qualified URL. If you'd like to use an
+image asset that is being uploaded as part of your blog post, the pattern for the
+final URL is as follows:
+
+  `<rackspace CDN>/<image name>-<sha256 hash>.<image extension>`
+
+Where:
+
+  * `<rackspace CDN>` is `https://657cea1304d5d92ee105-33ee89321dddef28209b83f19f06774f.ssl.cf1.rackcdn.com`
+  * `<image name>` is the case-sensitive name of the image _not including the extension_.
+  * `<sha256 hash>` is the 64-character hex output from running the command `sha256sum /path/to/image`
+  * `<image extension>` is `jpg`, `png`, etc.
+
+Example:
+
+  * `https://657cea1304d5d92ee105-33ee89321dddef28209b83f19f06774f.ssl.cf1.rackcdn.com/default-og-image-46fb3587dedfdf950188fabbddd596d67e6b699374a7f4e36b43046d7a24fd09.jpg`
+
+If you'd like to use an image asset from your post that _shouldn't_ appear in
+the post itself, you can include `style="display: none"` on the `<img />` tag
+to hide it within the post while triggering the necessary plugin code to ensure
+it is made available on the CDN based on the preceding naming convention.
+
+Make sure that the dates in the file name and front-matter match.
+
+Available categories include the following:
+
+- Ansible
+- Architecture
+- Automation
+- AWS
+- Azure
+- Chef
+- Cloud Files
+- Cloud Monitoring
+- Cloud-networks
+- Cloud Servers
+- Configuration Management
+- Database
+- Developers
+- DevOps
+- Docker
+- Events
+- General
+- Java
+- Jclouds
+- Jenkins
+- Mailgun
+- Neutron
+- NodeJS
+- OpenStack
+- Oracle
+- Orchestration
+- OSAD
+- Private Cloud
+- Python
+- Salesforce
+- SDK
+- Security
+- SQL Server
+
+If no category fits, use *General*.
+
+If you'd like to use a category that is not in the list, please send an email
+to infodev@rackspace.com. To avoid being flooded with categories, which might apply
+to only one or two blogs, we have automated throttling. However, notify us so
+that we can discuss your ideas for a new category.
+
+##### Excerpt
+
+Include an excerpt marker after your first paragraph or so to separate the
+preview text that appears on the blog index page from the full article. To do
+so, use the following HTML comment:
+
+```
+The excerpt paragraph, which should give teh reader a taste of what's to come.
+
+<!-- more -->
+
+The rest of your article.
+```
+
+The marker comment ``<!-- more -->`` must be on its own line, starting at
+column 1, and separated from content on either side by a single blank line.
+
+##### Images
+
+**To include images in your blog**, place them  within their respective blog
+directory. Within your blog, use the following markup:
+
+```markdown
+![Alt text here]({% asset_path YYYY-MM-DD-title-of-your-post/filename.png %})
+```
+
+**To wrap text around an image on the right**, use code similar to the following:
```

Comment:
this section needs to be revised for Hugo

Reply:
I believe this is the syntax {{- $image := resources.Get "YYYY-MM-DD-title-of-your-post/filename.png"-}} @robb-romans or @codebyravi can you confirm?

Reply:
I don't think you need all that, just `./image-filename.png` but I defer to @codebyravi